services/brig/src/Brig/Options.hs

```diff
@@ -627,6 +636,18 @@ defaultNonceTtlSecs = NonceTtlSecs $ 5 * 60 -- 5 minutes
 setNonceTtlSecs :: Settings -> NonceTtlSecs
 setNonceTtlSecs = fromMaybe defaultNonceTtlSecs . setNonceTtlSecsInternal
 
+defaultDpopMaxSkewSecs :: Word16
+defaultDpopMaxSkewSecs = 1
+
+setDpopMaxSkewSecs :: Settings -> Word16
+setDpopMaxSkewSecs = fromMaybe defaultDpopMaxSkewSecs . setDpopMaxSkewSecsInternal
+
+defaultDpopTokenExpirationTimeSecs :: Word64
+defaultDpopTokenExpirationTimeSecs = 5 * 60 -- 5 minutes
```

Comment:
i suppose seconds is fine here, but the default should be 30 seconds (or less?).  it's an automatic process, no human reaction times in the picture.